### DISCUSSION TITLE
help with pipenv --sync and --categories

### DISCUSSION THREAD

**Discussion by EiJayK** [pypa/pipenv#5577] [Q&A] [answered]
I noticed that when I use pipenv --sync it only installs packages specified in the .lock file under "default". Adding --dev installs also the development packages. However you must specify specific category names in order to get them installed when using --categories. Yet the official documentation mentions:

sync          Installs **all** packages specified in Pipfile.lock.

Am I overlooking something, or is this a shortcoming?

A nice feature would be something like this:

pipenv --sync --categories="*" indicating to install **all** category packages + the default.

Just an idea :-)

Have a nice evening.

**Comment by matteius** [comment, marked as answer]
It's a short coming of the documentation.  Will consider your proposal, another idea I've had is specifying the default categories for your system in a .env file so sync and lock or install would know your default preference.  Thoughts?

**Reply by EiJayK** [reply]
I think both will work, but "*" requires less typing 😃 Thanks for considering it 👍
